apps/storybook-react-native/src/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { TouchableOpacity, Text, StyleSheet } from 'react-native';
+
+export type MyButtonProps = {
+  onPress: () => void;
+  text: string;
+};
+
+export const MyButton = ({ onPress, text }: MyButtonProps) => {
+  return (
+    <TouchableOpacity>
+      <Text>{text}</Text>
+    </TouchableOpacity>
+  );
+};
```

Comment:
non-blocking: doesn't look like this file is being used

Reply:
will be for storybook. just setting it up right now